client/src/components/Ticketing/ticketingmanager/showings/ShowingUpdated/TicketTypeUpdateTable.tsx

```diff
@@ -0,0 +1,191 @@
+import {Field, useField} from 'formik';
+import React, {useState} from 'react';
+import {TicketTypeSelect} from './TicketTypeSelect';
+import {InputControl} from './InputControl';
+import {IconButton} from '@mui/material';
+import {useEvent} from './EventProvider';
+import {getTicketTypePrice} from './ShowingUtils';
+
+interface TicketTypeTableProps {
+  arrayHelpers;
+  eventInstanceID: number;
+}
+
+export const TicketTypeUpdateTable = (props: TicketTypeTableProps) => {
+  const {arrayHelpers, eventInstanceID} = props;
+  const {ticketTypes, showPopUp} = useEvent();
+  const [field] = useField('instanceTicketTypes');
+  const [availableTypes, setAvailableTypes] = useState(
+    ticketTypes
+      .filter(
+        (type) =>
+          !field.value.find((value) => value.typeID === Number(type.id)) && Number(type.id) !== 1,
+      )
+      .map((value) => Number(value.id)),
+  );
+
+  return (
+    <div
+      className={
+        'overflow-y-auto overflow-x-auto col-span-12 min-[1350px]:col-span-7 shadow-xl mx-auto rounded-xl bg-white w-[100%] min-h-[100px]'
+      }
+    >
+      <table className={'table table-fixed text-sm min-w-[100%]'}>
+        <thead
+          className={`text-left text-zinc-800 whitespace-nowrap bg-gray-300 ${
+            showPopUp ? '' : 'sticky z-50'
```

Comment:
The `z-index` needs to be below 10 for the edit version of the table header since the overall nav has a z-index of 10. Im getting this functionality lol
![image](https://github.com/WonderTix/WonderTix/assets/92124260/d8578d78-ab01-4634-a0f8-7e3427e970b5)